activesupport/test/core_ext/hash_ext_test.rb

```diff
@@ -963,8 +963,12 @@ def test_except_with_original_frozen
 
   def test_except_with_mocha_expectation_on_original
```

Comment:
Not really, what we're really testing is that except doesn't delete values. I'd write it as: `test_except_does_not_delete_values_in_original`